execution_engine/src/storage/trie_store/operations/debug_store.rs

```diff
@@ -0,0 +1,54 @@
+use std::marker::PhantomData;
+
+use casper_hashing::Digest;
+use casper_types::bytesrepr::{self, FromBytes};
+
+use crate::storage::{store::Store, trie::Trie, trie_store::TrieStore};
+
+/// A [`TrieStore`] wrapper that panics in debug mode whenever an attempt to deserialize [`V`] is
+/// made, otherwise it behaves as a [`TrieStore`].
+///
+/// The debug panic is used to ensure that this wrapper has  To ensure this wrapper has zero
```

Comment:
Some copy&paste error in the comment probably.

Reply:
Fixed